updateip.php

```diff
@@ -0,0 +1,221 @@
+<?php
```

Comment:
Hmm, why so complicated using username/password, re-creating the whole login-process (you dont check for locked users btw).
Just make this script "external" and POST to /api.php using apikeys and call API externally like in the examples here: https://github.com/Froxlor/Froxlor/blob/master/doc/example/list_functions.php

Reply:
The script can't be external because it performs authorization. It checks if a customer is allowed to update his subdomains. As discussed in IRC the dynamic DNS feature is not a wrapper of the API but a separate feature. It should be available without allowing the customer to change any DNS entry he likes (like MX, NS, etc.).

Should I refactor the user authentication code to be able to reuse it? Or should I refactor the API authentication code to be able to use it in the script? Or should we talk in IRC?

Reply:
Let's better talk on IRC...this might take a while ;)